extensions/grpc/deployment/src/test/java/io/quarkus/grpc/server/devmode/GrpcDevModeTest.java

```diff
@@ -126,6 +128,28 @@ public void testProtoFileChangeReload() throws InterruptedException {
         callHello("HACK_TO_GET_STATUS_NUMBER", "15");
     }
 
+    @Test
+    public void testProtoFileChangeAndImplementationReload() throws InterruptedException {
```

Comment:
@aloubyansky this test checks if the compilation order is correct, that is https://github.com/quarkusio/quarkus/pull/23271

Reply:
Thanks!